tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -592,6 +592,7 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         if (activeDashboards && selectedDashboard == null) {
           selectedDashboard = activeDashboards[0] || null;
           if (selectedDashboard != null) {
+            tf_storage.setString(tf_storage.TAB, selectedDashboard, /*useLocalStorage=*/false, /*useLocationReplace=*/true);
```

Comment:
Could you add a comment that explains why we're using location.replace here, i.e. to ensure that navigating back doesn't result in a loop?  Also maybe note that the call to tf_storage.setString() in this case will supersede the one that will be invoked by _selectedDashboardChanged() once we update _selectedDashboard below.

Also, this line is a bit long, can you wrap it e.g. to

```js
tf_storage.setString(
  tf_storage.TAB,
  selectedDashboard,
  /*useLocalStorage=*/false,
  /*useLocationReplace=*/true);
```

Reply:
HTML files should follow 100 char limit.